templates/init-script-cm.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: frontend-init-script
+  namespace: {{ .Values.environment }}
+data:
+  init-script.sh: |
+    #!/bin/bash
+
+    # Install jq if not available
+    JQ_URL=https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
+    JQ_BIN=/usr/local/bin/jq
+    which jq 2>&1 > /dev/null || { curl -sfL $JQ_URL -o $JQ_BIN && chmod +x $JQ_BIN ; }
+
+    # Generate settings.php from template
+    cd /patched-files
+    curl https://raw.githubusercontent.com/oda-hub/frontend-chart/master/config/drupal7_sites_default_settings.php.template -o settings.php
```

Comment:
this only makes sense for staging deployment from master. Also it excludes existience of deployments from master which are not updated with the last version from github.

could it be instead a file from a configmap?